### PR TITLE
Refactor human/Life()

### LINKED ISSUE #4849: Human Health Updates are Non-Instant
This was an issue with simple animals; the result of which, it made a lot of mobs far more durable than they otherwise should have been....

In any event, basically, when you take damage, your health *should* update **instantly**. That said, it does not, and is dependent on the next `life` process to catch up and update it.

This isn't terribly problematic in most cases, but it does lead to some scenarios (mostly with high damage weapons) whereby you don't lapse into crit instant (or die) the very moment you cross a damage threshold.

Would be fine if we were running something like gooncrit, as it's designed for that, but the current system is meant for instantaneous updates.

### LINKED ISSUE #4932: Changeling Lesser Form/SE Block 56 UI Issues.
**Problem Description**:
What is the problem?
Should a Changeling use lesser form, they lose the entirety of their UI, they can press F12 to partly restore it, however they cannot wear/remove clothes, gear, backpacks, and so on, severely crippling them mechanically.

Should they return to their human form, their humanUI does not return. F12 can partly fix this, but they remain unable to equip/use gear without hotkeys. I believe this is a significant problem.

This problem has also cropped up with monkey SE syringes and human SE syringes, the UI is fucked, and the only way to fix it is a brain transplant or cloning.

**What did you expect to happen**: 
Why do you think this is a bug?

I would expect the changeling or SE'd person in question to be able to properly equip themselves and use their gear.

**What happened instead**:
How is what happened different from what you expected?

Their UI is functionally permanently removed,

**Why is this bad/What are the consequences:**
Why do you think this is an important issue?

**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.
Get a player. Put it in a humanmob. Give them a monkey SE. Their UI is completely gone. Give them a human SE. Their UI remains completely gone. 

Get a player. Put it in a lessermob(monkey/farwa and so on), give it a human SE. UI is now gone. Give them a monkey SE. UI remains gone.

Get a changeling. Have them lesserform. UI is gone. Have them humanform. UI remains gone.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Anything else you can tell us.

Nope. This about covers it.

### PR DESCRIPTION
**Pull request by tigercat2000**
This refactors human/Life() to remove some of the styling atrocities going
on, and normalizes a number of gross procs like isInCrit(),
handle_mutations_and_radiation(), and handle_drunk().

Fixes #4849
Fixes #4932

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -3,1216 +3,1218 @@
...
-						updatehealth()
-
-					if(75 to 100)
-						radiation -= 3
-						adjustToxLoss(3)
-						damage = 3
-						if(prob(1))
-							to_chat(src, "<span class='alert'>You mutate!</span>")
-							randmutb(src)
+mob
```
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/life.dm]: This is relatively pathed.

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -3,1216 +3,1218 @@
...
+
+					handle_heartbeat()
+						var/client/C = src.client
+						if (C && C.prefs.sound & SOUND_HEARTBEAT) //disable heartbeat by pref
+							var/obj/item/organ/internal/heart/H = get_int_organ(/obj/item/organ/internal/heart)
+
+							if (!H) //H.status will runtime if there is no H (obviously)
+								return
+
+							if (H.status & ORGAN_ROBOT) //Handle robotic hearts specially with a wuuuubb. This also applies to machine-people.
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/life.dm]: `if (H:status & ORGAN_ROBOT) would be better. Come on.

File: code/modules/mob/living/carbon/human/life.dm (hunk 1 above)
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/life.dm]: is it

Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/life.dm]: Oh, blast, I've been bamboozled.

**Comment by Tastyfish** [comment]
kek

**Comment by TheDZD** [comment]
I think this PR has lived for long enough.

**Comment by PJB3005** [comment]
I was expecting a changelog.

Welp.

**Comment by KasparoVy** [comment]
Holy shit that was so dank

**Pull request closed without merging**